documentation/adoc/metrics.adoc

```diff
@@ -1,149 +1,137 @@
-# Metrics
+== Metrics
```

Comment:
Technically, this is not really about the Kafka metrics but about deploying Prometheus and Grafana. This is something we are not responsible for - this is more a guide if you want to try it and have no clue. So I would mark it as appendix with `[appendix]` and put it at the end of the document.